From 9cac75b662f23724251760b72c1ce1b6812ac143 Mon Sep 17 00:00:00 2001
From: Zhu Yuqing <b48954@freescale.com>
Date: Wed, 9 Jul 2014 11:18:17 +0800
Subject: [PATCH 072/115] ENGR00319919 fix OTA: The Upgrade button was cut off
 if enable font size as huge

enlarge the height of OTA dialogue. change layout_height from 178dp to 198dp

Signed-off-by: Zhu Yuqing <b48954@freescale.com>
---
 FSLOta/res/layout/main.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/FSLOta/res/layout/main.xml b/FSLOta/res/layout/main.xml
index 7de1107..09adf57 100644
--- a/FSLOta/res/layout/main.xml
+++ b/FSLOta/res/layout/main.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="wrap_content"
-    android:layout_height="178dp"
+    android:layout_height="198dp"
     android:baselineAligned="false"
     android:gravity="center|center_horizontal|center_vertical"
     android:orientation="horizontal" >
-- 
1.8.0

